### DISCUSSION TITLE
feature: Provide a way to list all compiler errors/page through them.

### DISCUSSION THREAD

**Discussion by jsatk** [LazyVim/LazyVim#3038] [Q&A]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

I often want to see _all_ of my compiler errors and jump through all the files with errors in them.

### Describe the solution you'd like

Some sort of way to see in the quickfix list or other menu all files with LSP errors.

### Describe alternatives you've considered

I primarily work in scala so I typically run `sbt compile` to generate the list of files w/ errors and then manually go to them in nvim.

### Additional context

There might already be a solution for this embedded in one of the many plugins that come with LazyVim. But if there is I have struggled to find it.

**Comment by dpetka2001** [comment]
`:h 'makeprg'` is what you need I believe along with `:h errorformat`.
